.mailmap

```diff
@@ -204,6 +204,7 @@ George Adams <gadams@microsoft.com> <george.adams@uk.ibm.com>
 Gerhard Stöbich <deb2001-github@yahoo.de>
 Gibson Fahnestock <gibfahn@gmail.com> <gib@uk.ibm.com>
 Gil Pedersen <git@gpost.dk> <github@gpost.dk>
+Giovanni Bucci <github@puskin.it>
```

Comment:
Out of curiosity, why this addition?

Reply:
Otherwise the inactive tool was flagging it

Reply:
Ah right, it must the name that mismatches (commit author is `Giovanni <github@puskin.it>`, while the tool is looking for `Giovanni Bucci <github@puskin.it>`)

Reply:
it is part of this doc: https://github.com/nodejs/node/blob/main/onboarding.md#exercise-make-a-pull-request-adding-yourself-to-the-readme . 


It says:  
```Run tools/find-inactive-collaborators.mjs. If that command outputs your name, amend the commit to include an addition to the [mailmap](https://github.com/nodejs/node/blob/main/.mailmap) file. See [gitmailmap](https://git-scm.com/docs/gitmailmap) for information on the format of the mailmap file.```